_posts/2019-11-13-cockpit-207.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Cockpit 207
+author: kkoukiou
+date: 2019-10-30
+tags: cockpit linux
+slug: cockpit-207
+category: release
+summary: Cockpit with authentication and storage page improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.
+
+### Cockpit's web server now accepts EC certificates
+
+To specify the [Elliptic Curve Cryptography](https://wiki.openssl.org/index.php/Elliptic_Curve_Cryptography) certificate
+the web service should use, simply drop a file with the extension .cert in the `/etc/cockpit/ws-certs.d` directory.
+
+### Storage: All Software Devices will be listed now in a single panel
+
+![Storage software devices](/images/storage-software-devices.png){:.screenshot}
+
+### Redesigned notifications
+
+This release of Cockpit features a new notification design.
+The new design corresponds to the [alert component from PatternFly 4](https://pf4.patternfly.org/components/Alert/examples/).
+
+![Alert firewall](/images/alert-firewall.png){:.screenshot}
+![Alert ipv4 settings](/images/alert-ipv4-settings.png){:.screenshot}
+![Alert diagnostic report](/images/alert-diagnostic-report.png){:.screenshot}
```

Comment:
I know you are proud of this (and you should be), but maybe one or two are enough :grin:

Reply:
Hehe sorry